src/users/containers/manage-user/manage-user.component.spec.ts

```diff
@@ -9,20 +9,25 @@ import * as fromRoot from '../../../app/store';
 import * as fromStore from '../../store';
 import * as fromOrgStore from '../../../organisation/store';
 import { MemoizedSelector } from '@ngrx/store';
-import { User } from '@hmcts/rpx-xui-common-lib';
+import { FeatureToggleService, User, UserAccessType } from '@hmcts/rpx-xui-common-lib';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { LoggerService } from 'src/shared/services/logger.service';
 import { OrganisationDetails } from 'src/models';
+import { EditUserModel } from 'src/user-profile/models/editUser.model';
 import { RpxTranslatePipe, RpxTranslationService } from 'rpx-xui-translation';
+import { StandardUserPermissionsComponent } from 'src/users/components/standard-user-permissions/standard-user-permissions.component';
+import { UserPersonalDetailsComponent } from 'src/users/components/user-personal-details/user-personal-details.component';
+import { AsyncPipe } from '@angular/common';
 
-describe('ManageUserComponent', () => {
+fdescribe('ManageUserComponent', () => {
```

Comment:
remove the f here else all the other tests won't run

Reply:
f removed to allow all tests to run